Sources/xcodes/main.swift

```diff
@@ -217,69 +229,103 @@ struct Xcodes: ParsableCommand {
             } else {
                 installation = .version(versionString)
             }
-            
+
             var downloader = XcodeInstaller.Downloader.urlSession
             if let aria2Path = aria2.flatMap(Path.init) ?? Current.shell.findExecutable("aria2c"),
                aria2Path.exists,
                noAria2 == false {
                 downloader = .aria2(aria2Path)
             }
-            
+
             let destination = getDirectory(possibleDirectory: directory)
-            
-            installer.install(installation, dataSource: globalDataSource.dataSource, downloader: downloader, destination: destination, experimentalUnxip: experimentalUnxip, emptyTrash: emptyTrash, noSuperuser: noSuperuser)
-                .done { Install.exit() }
-                .catch { error in
-                    Install.processDownloadOrInstall(error: error)
+
+            if select, case .version(let version) = installation {
+                firstly {
+                    selectXcode(shouldPrint: print, pathOrVersion: version, directory: destination, fallbackToInteractive: false)
```

Comment:
I think this is incorrect - we should install first, then select after.

Reply:
This was done to only select if it is already installed. If the version isn't installed this will throw and then we can install it.

Reply:
The new `select` flag on the install is `"Select the installed xcode version after installation."` which its not doing. 

You're checking if the select flag is passed in, trying to select the new xcode (that isn't installed) and then installing it.

Reply:
I was thinking of a use case where on CI you want to ensure that a certain Xcode version is installed. So you could do `xcodes install x.x.x --select --update`. If it's already installed you would expect it to be selected. 

So with all of these options if we try to select first, we exit super early by just selecting the version. But if we don't, it will first update the list, then see that it is already installed and select it. The list update will always happen even if Xcode was already installed and it wasn't needed. If that's the behavior you prefer I can remove it.